infra/main.bicep

```diff
@@ -799,26 +799,31 @@ module cosmosDb 'br/public:avm/res/document-db/database-account:0.6.1' = if (use
         containers: [
           {
             name: chatHistoryContainerName
+            kind: 'MultiHash'
```

Comment:
This probably will not be last revision to chat history, should we introduce a env var for version and set it on deployment so we know we need to run a migration?

Reply:
Currently we would know based off the container name. I could make that more explicitly look like a version like "chat-history-v1". I think that for migrating, you'd first run the provision, then run a migration script from the local machine, then do a full up/deploy.

Reply:
Or I can add this:

```
output AZURE_CHAT_HISTORY_VERSION string = 'cosmosdb-v2'
```

I don't actually need to set it on the app, as the app code only supports the v2 version, but a migration script could check that version.

Reply:
Actually we should set it on the app if we want to store it in the items as well. Adding.